BattleSim/src/main/java/com/revature/models/Roles.java

```diff
@@ -0,0 +1,81 @@
+package com.revature.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+
+/*CREATE TABLE roles (
+    r_id NUMBER(10) PRIMARY KEY,
+    name VARCHAR2(20));
+ * */
+
+@Entity
+public class Roles {
+
+	@Id
+	@Column(name="r_id", updatable=false)
+	@SequenceGenerator(sequenceName="role_id_maker", name="role_id_maker")
+	@GeneratedValue(generator="role_id_maker", strategy=GenerationType.SEQUENCE)
+	private int r_id;
+	
+	private String name;
+	
```

Comment:
id-less and no-arg constructors missing

Reply:
Verified that the two above constructors are added. Approved.